doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,144 @@
+"""
+=======================
+Background Subtraction using the Rolling Ball Method
```

Comment:
I personally prefer titles at the imperative ("Subtract background ...") because they are descriptive, direct, and engaging. Currently, there is no standard (https://scikit-image.org/docs/dev/auto_examples/index.html) so I'm not requesting a change in this PR, I just want to spark the discussion. @emmanuelle @jni

Reply:
I guess one counter-argument here is that "Background Subtraction" is a keyword, and might help with SEO.

Reply:
How about something like `(How to (?)) Use the Rolling Ball Method for Background Subtraction`?